### DISCUSSION TITLE
no force-blur option

### DISCUSSION THREAD

**Discussion by J-Corleone** [taj-ny/kwin-effects-forceblur#63] [Q&A] [answered]
I want to know how to use, why after `make install`, there's no _**Force Blur**_ option in _**desktop effects**_ ?
OS: Arch Linux x86_64
Kernel: 6.8.2-arch2-1
DE: Plasma 6.0.4
WM: kwin

![Screenshot_20240426_002656](https://github.com/taj-ny/kwin-effects-forceblur/assets/80458454/50422f92-9a7d-4962-8df6-ccde0791ccd6)
![Screenshot_20240426_002830](https://github.com/taj-ny/kwin-effects-forceblur/assets/80458454/8ca264cd-c347-4ad4-962d-4e2b80c28090)

**Comment by taj-ny** [comment, marked as answer]
Try restarting Plasma. If that doesn't work, remove the *build* directory and build the effect again.

**Reply by J-Corleone** [reply]
build again, works!
